source/_integrations/androidtv_remote.markdown

```diff
@@ -31,18 +31,21 @@ The entity has the `current_activity` attribute that shows the current foregroun
 You can use the `remote.turn_off`, `remote.turn_on`, `remote.toggle`,  and `remote.send_command` services from the [remote](/integrations/remote/) platform.
 
 For a list of the most common commands you can send to the Android TV via `remote.send_command` see: [TvKeys](https://github.com/tronikos/androidtvremote2/blob/main/TvKeys.txt).
-For a full list see [here](https://github.com/tronikos/androidtvremote2/blob/main/src/androidtvremote2/remotemessage.proto#L90).
+For a full list see [here](https://github.com/tronikos/androidtvremote2/blob/main/src/androidtvremote2/remotemessage.proto#L90) (note most of the keys in the full list typically don't work).
 
-If `activity` is specified in `remote.turn_on` it will open the specified URL in the associated app.
+If `activity` is specified in `remote.turn_on` it will open the specified URL in the associated app via [deep linking](https://developer.android.com/training/app-links/deep-linking).
 
 Examples of URLs to pass as activity for some popular apps:
 
 | App | URL |
 | --- | --- |
-| YouTube | https://www.youtube.com
-| Netflix | https://www.netflix.com/title
-| Prime Video | https://app.primevideo.com
-| Disney+ | https://www.disneyplus.com
+| YouTube | `https://www.youtube.com` or `vnd.youtube://` or `vnd.youtube.launch://`
+| Netflix | `https://www.netflix.com/title`
+| Prime Video | `https://app.primevideo.com`
+| Disney+ | `https://www.disneyplus.com`
+| Plex | `plex://`
+
+See [this Guide/wiki](https://community.home-assistant.io/t/android-tv-remote-app-links-deep-linking-guide/567921) for how to find out what URL to use and a collection of more apps.
```

Comment:
Why is that a community guide?

If it matter for the core integration, it should be part of the documentation.

Reply:
I don't feel the linked guide should be part of this documentation. I think it's enough to say here the integration accepts deep links. How to figure them out and having a complete list of them is outside the scope of this documentation.

Reply:
We don't support community guides (hence the name/location).

The fact we need to point to there, indicates our main documentation is not sufficient. We need to extend the documentation in that case, not link to other community resources to resolve the problem.

../Frenck

Reply:
I removed the link to the guide for now.